docs/source/v2/tenants.blade.md

```diff
@@ -87,7 +87,7 @@ You may also use `->with<PropertyNameInPascalCase>` and it will be stored in sna
 These methods make the most sense (= sound the best) during tenant creation:
 
 ```php
-// $domains = ['foo.yourapp.com', 'foo.com'];
```

Comment:
I kept this commented to just hint what's in that variable. In real world, you wouldn't define it, it would come from a request or something.

Perhaps there's some clearer way to express that?

Reply:
Fair point.